service/model/project.go

```diff
@@ -22,10 +22,17 @@ type Project struct {
 	SlackChannelID            string
 	ReleaseNotificationConfig ReleaseNotificationConfig
 	GithubRepositoryURL       url.URL
+	GithubRepo                GithubRepo
 	CreatedAt                 time.Time
 	UpdatedAt                 time.Time
 }
 
+type GithubRepo struct {
+	HTMLURL   url.URL
```

Comment:
I would also keep the URL in the struct because the user will provide a URL. When passing the project back to the transport layer, it will be good to also return the URL so the client can show it to the user (as they won't care about slugs).